features/bootstrap/DigestContext.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+use eLife\ApiSdk\ApiSdk;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+
+final class DigestContext extends Context
+{
+    private $numberOfDigests;
+
+    /**
+     * @BeforeScenario
+     */
+    public function setFeatureFlagCookie()
+    {
+        putenv('FEATURE_DIGEST_CHANNEL=true');
```

Comment:
given some scenarios are `@javascript` and run through a headless browser, this environment variable should have no effect on them as it's a different process answering HTTP requests? Setting the cookie should be enough and should scale with parallel scenarios being tested simultaneously in `ci`?

Reply:
So the two lines here are alternatives to each other (non-@javascript talks to the kernel, @javascript goes through some sort of browser)? If so two comments or two private methods explaining what they do can avoid a future headache